plugins/smart_proxy_discovery/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-smart-proxy-discovery (1.0.3) stable; urgency=low
```

Comment:
I don't know if @lzap used it, but I think the update script I contributed the other week leaves off the `-1`.  I've always been a little unclear about whether to include it or not for the first package of a new upstream version, but on re-reading [this bit of the Debian manual](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Version) (actually describing the binary control file), it's convention to include it.